backend/src/agents/graph_schema.py

```diff
@@ -0,0 +1,19 @@
+narrative_options = "Bills, Groceries, Entertainment, Rent, Shopping"
+
+graph_schema = f"""
+Node properties are the following:\n
```

Comment:
Up to you, but our other prompts are being stored as `.j2` templates. Aware this isn't a template and more of a long string but as it is used as a prompt may be appropriate in the `/prompts` directory

Reply:
Good idea, I've moved it as well